ansible/roles/software/ckan/security-fix/tasks/main.yml

```diff
@@ -10,3 +10,21 @@
   with_items:
     - /etc/ckan
     - "{{ project_source_new_code_path }}"
+
+- name: Fix 'Content-Cache' header
+  replace:
+    dest: "{{ project_source_new_code_path }}/src/ckan/ckan/lib/base.py"
+    regexp: ' = "private"'
```

Comment:
if copy the whole fine, it is easy to lose track what and why the change we made over the defaults. when doing replace I would prefer including more text into the regex phrase. It is a file with a couple of hundred lines and it might contains multiple occurrences of ` = "private"` in the future version.

so i prefer lineinfile with more defined regex phrase.